worker/migrations/migrations.go

```diff
@@ -80,6 +91,137 @@ func commit(ctx *job.WorkerContext, err error) error {
 	return err
 }
 
+// Migrate all the encrypted accounts to Bitwarden ciphers.
+// It decrypts each account, reencrypt the fields with the organization key,
+// and save it in the ciphers database.
+func migrateAccountsToOrganization(domain string) error {
+	inst, err := instance.GetFromCouch(domain)
+	if err != nil {
+		return err
+	}
+	setting, err := settings.Get(inst)
+	if err != nil {
+		return err
+	}
+	// Get org key
+	var orgKey []byte
+	orgKey, err = setting.OrganizationKey()
+	if err == settings.ErrMissingOrgKey {
+		// Create org key if missing
+		if err := setting.EnsureCozyOrganization(inst); err != nil {
+			return err
+		}
+		orgKey, err = setting.OrganizationKey()
+		if err != nil {
+			return err
+		}
+	} else if err != nil {
+		return err
+	}
+
+	// Iterate over all triggers to get the konnectors with the associated account
+	jobsSystem := job.System()
+	triggers, err := jobsSystem.GetAllTriggers(inst)
+	if err != nil {
+		return err
+	}
+	var msg struct {
+		Account string `json:"account"`
+		Slug    string `json:"konnector"`
+	}
+	for _, t := range triggers {
+		if t.Infos().WorkerType != "konnector" {
+			continue
+		}
+		err := t.Infos().Message.Unmarshal(&msg)
+		if err == nil && msg.Account != "" && msg.Slug != "" {
```

Comment:
In go, we often prefer to use shortcuts instead of long `if` branches. Here, we can do:

```go
if err != nil || msg.Account == "" || msg.Slug == "" {
    continue
}
```